api/api.go

```diff
@@ -161,6 +161,12 @@ func (api *API) setupRoutes() {
 	// V1 API
 	v1 := api.r.Group("/api/v1")
 
+	// system checks used to verify the integrity of our services
+	systemChecks := v1.Group("/systems")
```

Comment:
just to confirm, these APIs should be accessible to anyone right?

Reply:
Yup! It's only intended to give a *very* basic response. Ie, status 200 and a basic "systems online" as the response.